### DISCUSSION TITLE
Windows 10 style Start menu doesn't show

### DISCUSSION THREAD

**Discussion by nethew** [valinet/ExplorerPatcher#1496] [Q&A] [answered]
First and foremost thank you very much for ExplorerPather. I have recently installed Win11 on a new PC and couldn't do without it.

When I set start menu Style to Windows 10, the menu never shows (on click or ctrl+Esc) no matter what other setting I change.
The menu probably opens, because the current application loses focus and if I type calc and press Enter the Calculator is sometimes started.
I have a single monitor setup.

Also what does the Pin tiles to Windows 10 Start menu from File Explorer link do? It only opens File Explorer on This PC. What should I do from there?

**Comment by pyrates999** [comment, marked as answer]
The symbols have not been downloaded I think:

https://github.com/valinet/ExplorerPatcher/wiki/Symbols

What version of windows 11 are you running?  You can find out by running winver.  Versions with preview updates installed and insider builds are not supported because Microsoft doesn't make symbols available for those versions.

Are you running any third party firewall software?  If so, try disabling them as they could be preventing the symbols from being downloaded in the background.

**Reply by nethew** [reply]
Thank you.

Windows version is 22H2, build 22621.1178
ExplorerPatcher is 22621.1555.55.2

The day before yesterday I have installed 2023-05 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5026446) so that's probably it. I'm not sure if it worked before the update, because I have started using ExplorerPatcher after that.

Does it mean that if I wait for the update preview to become a normal update, the symbols will update and the start menu will then work? Or do I have to do something?

**Reply by nethew** [reply]
I have now tried uninstalling ExplorerPatcher (for some other reason) and when I installed it back, the Windows 10 style Start menu now shows ok.

**Reply by pyrates999** [reply]
I would uninstall any preview updates and don't install any preview updates in the future if you want to use explorer patcher.

**Reply by nethew** [reply]
Ok, thank you. I'll be careful about them in the future.
